### ISSUE TITLE
Expose version etc from Visual Studio Marketplace badges

### ISSUE THREAD

**Issue by mwpowellhtx** [badges/shields#10246]
:clipboard: I see there is a [_Visual Studio Marketplace_](https://shields.io/badges/visual-studio-marketplace-downloads) badge, which in of itself is great. However, I am not sure the metric being indicated, is that a number of downloads, I'm not sure.

That being said, would be great if fields like version number, title, etc, could be exposed. That would make more sense, but I do not see those options enumerated among any of the required or optional features. Along similar lines as [_NuGet badges_](https://shields.io/badges/nu-get-version) exposes.

**Comment by mwpowellhtx** [comment]
To clarify, we do not care about, presumable, downloads or subscribers or what not, or even package size, per se, whatever the case may be. But we would like to know the deployed version, if at all possible.

![image](https://github.com/badges/shields/assets/1824187/6df4b5a9-f931-44bb-bba6-142db31eee02)

In [_this particular instance_](https://marketplace.visualstudio.com/items?itemName=PrecisionInfinity.AutomaticVersions), we would expect something like `2.1.2`.

![image](https://github.com/badges/shields/assets/1824187/3672b2a6-008e-4ace-bfdf-70aabeba84ff)

**Comment by chris48s** [comment]
Yes. This shows number of downloads. That's why the title of the page you linked is "Visual Studio Marketplace Downloads" and it is in the "Downloads" category on the menu.

There is also a Visual Studio Marketplace Version badge: https://shields.io/badges/visual-studio-marketplace-version

![](https://img.shields.io/visual-studio-marketplace/v/PrecisionInfinity.AutomaticVersions
)
https://img.shields.io/visual-studio-marketplace/v/PrecisionInfinity.AutomaticVersions

**Comment by mwpowellhtx** [comment]
Ah! Did not know that. Good to know, thanks!
